docs/README.md

```diff
@@ -1,40 +1,51 @@
-# Luigi Documentation
+# Luigi Documentation
 
-The aim of this documentation is to provide step-by-step instructions for Luigi as well as to provide a comprehensive reference of all the features you can use. 
+If you are new to Luigi, read the [Getting started guide](getting-started.md) to learn more about its structure and key functions.
 
-[**Luigi Core**](#luigi-core) refers to the main application in Luigi. The Luigi Core libraries allow you to create a user interface where you can configure navigation, authorization, and more. 
+## Luigi Core
 
-[**Luigi Client**](#luigi-client) refers to the micro frontends or views which the Luigi Core application contains. The Luigi Client API allows you to configure settings which ensure smooth communication between Core and Client.
+Luigi Core enables you to create a web application with an easily configurable and consistent user interface.
 
-## Luigi Core
+[Installation](application-setup.md) shows you the first steps to prepare your application for development.
 
-Set up and configure your main Luigi application by following these guides. 
+### Configuration
 
-* [Application setup](application-setup.md) shows you the first steps to prepare your application for development.
-* [Navigation configuration](navigation-configuration.md) shows you how to configure navigation in Luigi.
-* [Navigation parameters reference](navigation-parameters-reference.md) lists the parameters you can use to configure the top and side navigation.
-* [Authorization configuration](authorization-configuration.md) guides you through the configuration to secure Luigi.
-* [Authorization events](authorization-events.md) guides you through the event configuration to react to Luigi authorization events.
-* [General settings](general-settings.md) provides you with additional configuration parameters for Luigi.
-* [Luigi lifecycle hooks](lifecycle-hooks.md) allows you to execute custom logic on any of the Luigi lifecycle steps.
-* [Core API](luigi-core-api.md) provides you with API features that help you enrich and use Luigi Core.
-* [Luigi UI features](luigi-ux-features.md) provides different customization options for some of the Luigi UI components.
+> **TIP:** The [Luigi Fiddle](fiddle.luigi-project.io) allows you to configure a very simple application and get a feel for the process.
```

Comment:
The link doesn't work currently. Probably it has to be https://fiddle.luigi-project.io